app/static/js/event_list.js

```diff
@@ -41,3 +41,16 @@ function removeRsvpForEvent(eventID){
 
   })
 }
+
+document.getElementById("toggleButton").addEventListener("click", function() {
+  var tableRows = document.getElementsByClassName("showlist");
+
+  for (var i = 0; i < tableRows.length; i++) {
+    var tableRow = tableRows[i];
+    if (tableRow.classList.contains("d-none")) {
+      tableRow.classList.remove("d-none");
+    } else {
+      tableRow.classList.add("d-none");
```

Comment:
Use .hide() and .show() on tablerow instead of adding and removing `d-none` (you will need to use `display:none` in the html instead of the `d-none` class)